packages/wallet-sdk/src/sign/scw/transport/ConfigMessage.ts

```diff
@@ -3,24 +3,40 @@ import { Message } from ':core/communicator/Message';
 export interface ConfigMessage extends Message {
   type: 'config';
   event: {
-    type: ClientConfigEventType | HostConfigEventType;
+    type: ConfigEventType;
     value?: unknown;
   };
 }
 
-export enum ClientConfigEventType {
-  SelectConnectionType = 'selectConnectionType',
-  DappOriginMessage = 'dappOriginMessage',
-  WalletLinkUrl = 'walletLinkUrl',
-  WalletLinkQrScanned = 'walletLinkQrScanned',
+export type ConfigEventType =
+  | PopupSetupEventType
+  | SignerConfigEventType
+  | WalletLinkConfigEventType;
```

Comment:
nice structure.
let's revisit the names though.
would be nice if they look consistent, and easy to understand who's sender and receiver and logical sequence of them.
take a look at things like "ssl handshake process". might give some inspirations?

and maybe the whole structure of this type 
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/6d563eb6-908e-449b-a06b-748b20d48186)
can it be just
type: 'config'
message: ConfigEventType
params?: unknown
?

Reply:
let's revisit this. how can we make this more organized? 
don't have a solution myself for now, but it doesn't look optimal anyway